typing-break/drwright.c

```diff
@@ -866,18 +901,22 @@ drwright_new (void)
 		setup_debug_values (dr);
 	}
 
-	dr->ui_manager = gtk_ui_manager_new ();
-
-	action_group = gtk_action_group_new ("MenuActions");
+	ui_builder = gtk_builder_new ();
 #ifdef ENABLE_NLS
-	gtk_action_group_set_translation_domain (action_group, GETTEXT_PACKAGE);
+	gtk_builder_set_translation_domain (ui_builder, GETTEXT_PACKAGE);
 #endif /* ENABLE_NLS */
-	gtk_action_group_add_actions (action_group, actions, G_N_ELEMENTS (actions), dr);
-	gtk_ui_manager_insert_action_group (dr->ui_manager, action_group, 0);
-	gtk_ui_manager_add_ui_from_string (dr->ui_manager, ui_description, -1, NULL);
+	gtk_builder_add_from_string (ui_builder, ui_description, -1, NULL);
+	dr->menu = (GtkWidget *)gtk_builder_get_object (ui_builder, "pop_menu");
```

Comment:
You need to hold a reference to the menu, otherwise destroying the `GtkBuilder` below will also destroy the menu as it's not tied to anything. Currently the menu doesn't work for me, and I get:
```
(mate-typing-monitor:1578107): GLib-GObject-WARNING **: 11:53:54.367: invalid unclassed pointer in cast to 'GtkMenu'

(mate-typing-monitor:1578107): Gtk-CRITICAL **: 11:53:54.367: gtk_menu_popup: assertion 'GTK_IS_MENU (menu)' failed
```
when trying to open the status icon's menu.


```suggestion
	dr->menu = (GtkWidget *) g_object_ref (gtk_builder_get_object (ui_builder, "pop_menu"));
```